google/cloud/spanner/internal/spanner_stub_factory.cc

```diff
@@ -47,11 +47,14 @@ std::shared_ptr<SpannerStub> CreateDefaultSpannerStub(
   if (auth->RequiresConfigureContext()) {
     stub = std::make_shared<SpannerAuth>(std::move(auth), std::move(stub));
   }
-  stub = std::make_shared<MetadataSpannerStub>(std::move(stub), db.FullName());
+  stub = std::make_shared<SpannerMetadata>(
```

Comment:
Shouldn't this use the new `SpannerTracingStub` too?

Reply:
My intention was to have this PR be a pure refactor. The next PR will include the tracing stub:

https://github.com/googleapis/google-cloud-cpp/commit/742b02d40b119a9ea7db0542d87cd07a4a7ae412